app/src/views/GrantRegistryGrantDetail.vue

```diff
@@ -649,7 +653,8 @@ function useGrantDetail() {
     form.value.payee = grant.value?.payee || '';
     form.value.name = grantMetadata.value?.name || '';
     form.value.description = grantMetadata.value?.description || '';
-    form.value.logoURI = ipfs.getMetaPtr({ cid: grantMetadata.value?.logoPtr?.pointer }) || undefined;
+    const cid = grantMetadata.value?.logoPtr?.pointer;
```

Comment:
I'd personally move this above the `form.value.*` defs - but it's fine where it is!